client/src/pages/component/homeBody/Homebody.css

```diff
@@ -1,4 +1,88 @@
-.homebody{
-    text-align: center;
-    margin-top: 100px;
+.homebody-container {
+	background-image: url(../image/home-body.jpg);
+	background-repeat: no-repeat;
+	width: 100%;
+	height: 650px;
+	background-size: contain;
+	background-position: cover;
+}
+.body-main-container {
+	position: relative;
+	height: 100vh;
+}
```

Comment:
are these realy needed? I turned them off and nothing changed
<img width="1755" alt="Screenshot 2022-11-18 at 14 59 56" src="https://user-images.githubusercontent.com/65277182/202734464-f91ac87b-d809-49ab-b880-ff657bd6488f.png">